pipit/tests/trace.py

```diff
@@ -49,9 +49,9 @@ def test_match_events(data_dir, ping_pong_otf2_trace):
     # timestamps.  Compares the values of these lists to ensure the pairing
     # functions produced correct results.
     rank_0_indices = rank_0_df.index.to_list()
-    rank_0_matching_indices = rank_0_df["Matching Index"].to_list()
```

Comment:
all of the changes in these are files so that the column names reflect the recent change